services/apps/src/apps/routes/domain/permission.clj

```diff
@@ -0,0 +1,18 @@
+(ns apps.routes.domain.permission
+  (:use [common-swagger-api.schema :only [describe]]
+        [schema.core :only [defschema optional-key enum]])
+  (:import [java.util UUID]))
+
+(defschema AppIdList
+  {:apps (describe [String] "A List of app identifiers")})
+
+(defschema UserPermission
+  {:user       (describe String "The user ID")
+   :permission (describe String "The permission level assigned to the user")})
```

Comment:
It'd be better if this was an `s/enum` of the available permissions, if that's something we can expect to be able to know in advance.